tests/e2e/specs/wcpay/shopper/shopper-checkout-purchase-site-editor.spec.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+const { shopper, merchant } = require( '@woocommerce/e2e-utils' );
+
+/**
+ * Internal dependencies
+ */
+
+import {
+	fillCardDetails,
+	confirmCardAuthentication,
+	setupProductCheckoutNoMiniCart,
+} from '../../../utils/payments';
+
+import { shopperWCP, merchantWCP } from '../../../utils';
+
+const cardTestingPreventionStates = [
+	{ cardTestingPreventionEnabled: false },
+	{ cardTestingPreventionEnabled: true },
+];
+
+describe.each( cardTestingPreventionStates )(
+	'Successful purchase, site builder theme',
+	( { cardTestingPreventionEnabled } ) => {
+		beforeAll( async () => {
+			await merchant.login();
+			await merchantWCP.changeTheme( 'twentytwentyfour' );
+			if ( cardTestingPreventionEnabled ) {
+				await merchantWCP.enableCardTestingProtection();
+			}
+			await merchant.logout();
+		} );
+
+		beforeEach( async () => {
+			await setupProductCheckoutNoMiniCart(
+				config.get( 'addresses.customer.billing' )
+			);
+		} );
+
+		afterAll( async () => {
+			// Clear the cart at the end so it's ready for another test
+			await shopperWCP.emptyCart();
+			await merchant.login();
+			await merchantWCP.changeTheme( 'storefront' );
```

Comment:
```
await activateTheme( 'storefront' );
```

Reply:
Replaced in https://github.com/Automattic/woocommerce-payments/pull/8164/commits/8a7bbc244edff7f1723e103975ca10733fb3e23b. the suggestion!